content/applications/inventory_and_mrp/purchase/products/pricelist.rst

```diff
@@ -0,0 +1,259 @@
+
+Below is a
+list of commonly used fields when importing vendor pricelists:
+
+.. list-table:: Field name definitions
+   :header-rows: 1
+
+   * - Field name
+     - Used for
+     - Field in Odoo database
+     - Technical name of field
+   * - Vendor
+     - The only required field for creating a vendor pricelist record. This field specifies the
+       vendor associated with the product.
+     - :guilabel:`Vendor` field in the :ref:`vendor pricelist of the product form
+       <purchase/products/pricelist>`.
+     - partner_id
+   * - Product Template
+     - The Odoo product the vendor pricelist entry is related to.
+     - :guilabel:`Product` field in the vendor pricelist.
+     - product_tmpl_id
+   * - Quantity
+     - The minimum quantity required to receive the product at the specified price.
+     - :guilabel:`Quantity` field in the vendor pricelist. (If not visible, enable it by clicking
+       the :icon:`oi-settings-adjust` :guilabel:`(settings)` icon and tick the :guilabel:`Quantity`
+       checkbox)
+     - min_qty
+   * - Unit Price
+     - The purchase price for the product from the vendor.
+     - :guilabel:`Price` field in the vendor pricelist.
+     - price
+   * - Delivery Lead Time
+     - :ref:`Number of days <inventory/shipping_receiving/purchase-lt>` before receiving the
+       product after confirming a purchase order.
+     - :guilabel:`Delivery Lead Time` field on the vendor pricelist.
+     - delay
+   * - Sequence
+     - Defines the order of vendors in the pricelist when multiple vendors are available. For
+       example, if `Azure Interior` is listed first and Wood Corner second, their sequences would be
+       `1` and `2`.
+     - N/A
+     - sequence
+   * - Company
+     - Company name that the product belongs to.
+     - :guilabel:`Company` field in the vendor pricelist.
+     - company_id
+   * - :ref:`External ID <purchase/products/external-id>`
+     - Unique ID of a record used to update existing vendor pricelists.
+     - N/A
+     - id
+
+Import records
+--------------
+
+With a template downloaded, fill out the XLSX or CSV file with the necessary information. After
+inputting everything, import the file back into the Odoo database by going to
+:menuselection:`Purchase app --> Configuration --> Vendor Pricelists`.
+
+On the page, click the :icon:`fa-cog` :guilabel:`(gear)` icon in the top-left corner. In the
+drop-down menu that appears, click :guilabel:`Import records`.
+
+Then, click :guilabel:`Upload File`, and after selecting the Excel or CSV file, confirm the correct
+fields, and click :guilabel:`Import`.
+
+.. seealso::
+   - :doc:`../../../essentials/export_import_data`
+   - :ref:`Common column names <purchase/products/common-fields>`
+
+.. image:: pricelist/supplier-pricelist-example.png
+   :align: center
+   :alt: Upload file screen.
+
+Formatting import file
+~~~~~~~~~~~~~~~~~~~~~~
+
+To understand how to format imports for vendor pricelists, consider the following example.
```

Comment:
Consider rephrasing "how to format imports for" to "how to format import files for" or something like that! Just a nitpick.

Reply:
Great suggestion!